odoo/addons/base/models/ir_cron.py

```diff
@@ -50,7 +50,7 @@ class ir_cron(models.Model):
     ir_actions_server_id = fields.Many2one(
         'ir.actions.server', 'Server action',
         delegate=True, ondelete='restrict', required=True)
-    cron_name = fields.Char('Name', related='ir_actions_server_id.name', store=True, readonly=False)
+    cron_name = fields.Char('Name', related='ir_actions_server_id.name', readonly=False)
```

Comment:
You cannot do that on a stable version.
Moreover, I don't see how it relates to the issue you are fixing.